sync usage from README (#99)
iterative/cml.dev: content/docs/usage.md

# Using CML

You'll need a GitLab, GitHub, or Bitbucket account to begin. Users may wish to
familiarize themselves with [Github Actions](https://help.github.com/en/actions)
or

> [GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration).

Reviewer: **You** versus **users**... who wins?

Author: lol neither would be my choice. "You" is far too informal, while "users" is too disinterestedly cold. Inconsistency is even worse.

```suggestion
A GitLab, GitHub, or Bitbucket account is required. Familiarity with
[Github Actions](https://help.github.com/en/actions) or
[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration)
may also be beneficial.
```

> Here, will discuss the GitHub use case.

Reviewer: This remark may be a bit outdated, given that we provide separate headings for each platform.

_Bitbucket Server support estimated to arrive by mid 2021._

## GitHub

The key file in any CML project is `.github/workflows/cml.yaml`:

```yaml
name: your-workflow-name
on: [push]
jobs:
  run:
    runs-on: ubuntu-latest
    # optionally use a convenient Ubuntu LTS + CUDA + DVC + CML image
    # container: docker://iterativeai/cml:0-dvc2-base1-gpu
    # container: docker://ghcr.io/iterative/cml:0-dvc2-base1-gpu
    steps:
      - uses: actions/checkout@v2
      # may need to setup NodeJS & Python3 on e.g. self-hosted
      # - uses: actions/setup-node@v2
      #   with:
      #     node-version: '12'
      # - uses: actions/setup-python@v2
      #   with:
      #     python-version: '3.x'
      - uses: iterative/setup-cml@v1
      - name: Train model
        run: |
          # Your ML workflow goes here
          pip install -r requirements.txt
          python train.py
      - name: Write CML report
        env:
          REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        run: |
          # Post reports as comments in GitHub PRs
          cat results.txt >> report.md
          cml-send-comment report.md
```

We helpfully provide CML and other useful libraries pre-installed on our

> [custom Docker images](/doc/self-hosted-runners#docker-images). In the above

Reviewer: Not for this pull request, but we should proably consider creating a separate section for [...] images, as per https://github.com/iterative/cml.dev/issues/86#issuecomment-900989038; they have more use cases than self-hosted runners.

Reviewer: Moved to https://github.com/iterative/cml.dev/issues/102
